package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bin-build",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
You shouldn't touch this; it's the maintainer's job/choice.

Reply:
Let alone, this should be a major bump due to the node.js version requirement bump.

Reply:
Thx for the advice. I have reverted that updated, and rebased.